src/talk/cmdline.rs

```diff
@@ -0,0 +1,35 @@
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+/// The talk command will send audio for the camera to say
+///
+/// This data should be encoded in a way that gstreamer can understand.
+/// This should be ok with most common formats.
+///
+/// `gst-launch` can be used to prepare this data
+#[derive(StructOpt, Debug)]
+pub struct Opt {
+    /// The path to the config file
+    #[structopt(short, long, parse(from_os_str))]
+    pub config: PathBuf,
+    /// The name of the camera to change the lights of. Must be a name in the config
+    pub camera: String,
+    /// The path to the audio file.
```

Comment:
Would it be possible to accept this via pipe/stdin from gstreamer directly? I think that prevents you from having to handle the microphone directly (unless that's a particularly desired feature) - just feed it from gstreamer

Reply:
I can certainly handle std-in data but then I am not sure what the format it will be in is.

Reply:
I think for now though lets just implement it in gstreamer then add std-in as a seperate PR. I think we should have both methods